devtools/etdump/utils.h

```diff
@@ -0,0 +1,24 @@
+// (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.
+
+#include <cstddef>
+#include <cstdint>
+
+#pragma once
+
+namespace executorch {
+namespace etdump {
+namespace internal {
+
+inline uint8_t* align_pointer(void* ptr, size_t alignment) {
```

Comment:
Please add a doc comment, and point out that this assumes that `alignment` is a power of 2.